05LessonFive/Checkpoint1/css/style.css

```diff
@@ -1,43 +1,30 @@
-.lawn {
-    width: 100%;
-    height: 300px;
+#sec1 {
+  width: 500px;
+  height: 400px;
+  background: no-repeat center/500px url(../img/berries.jpg);
+  margin-right: 10px;
+  float:left;
 }
-
-.lawn h1 {
-    text-align: center;
-    font-family: monospace;
-    color: darkgray;
+#sec2 {
+  width:500px;
+  height: 300px;
+  text-align: center;
+  padding-top: 100px;
+  background-color: rgb(240,240,240);
+  float:left;
 }
-
-.lawn {
-    background-image: url(../img/banner-garden-stmichaelsmount.jpg);
-    background-repeat: no-repeat;
-    background-position: center;
-    background-size: cover;
+h1 {
+  font-size: 1.9em;
+  color: #252b3b;
 }
-
-.left {
-    float: left;
-    width: 60%;
-    height: 300px;
-    margin-left: 260px;
-    background-color: lightgray;
+p {
+  color: #555;
+  padding-bottom: 20px;
 }
-
-.left h1 {
-    text-align: center;
+a {
+  text-decoration: none;
+  border: 2px solid #555;
+  padding: 5 px;
```

Comment:
the browser doesn't like spaces between the numeric values and the type of value. For example, the 5 px you have here wasn't picked up. It needs to be 5px.

Font-weight should also be in numeric values. 700 is bold and they increment in 100s.